docs/custom_ui_extensions/custom_hook.md

```diff
@@ -65,19 +66,26 @@ class Hook {
      * @param {string} mode - edit,create or clone
      * @param {object} util - the utility object
      */
-    constructor(globalConfig, serviceName, state, mode, util) {
+    constructor(globalConfig, serviceName, state, mode, util, groupName) {
```

Comment:
Is it going to break existing custom hooks if we add a new parameter here?

Reply:
This is the new parameter, it won’t break any existing functionality.